src/layouts/Authentication/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { useEffect } from 'react';
+
+const layoutClassName = 'layout-authentication';
+
+export const layoutAuthenticationTestIds = {
+  base: layoutClassName,
+};
+
+interface LayoutProps {
+  children: React.ReactNode;
+}
+
+const LayoutAuthentication = ({ children }: LayoutProps) => {
+  useEffect(() => {
+    document.body.classList.add(layoutClassName);
+  }, []);
```

Comment:
You might need this code in other layouts, should we create a hook and use it instead?
For example, you can make the hook `useLayout` and use it `useLayout(LayoutName)` 🤔 

But it can be added in the later task as you have only 1 layout for now